### DISCUSSION TITLE
Custom auto-type with Private URL tab in browser

### DISCUSSION THREAD

**Discussion by ElPadreX** [keepassxreboot/keepassxc#11240] [Ideas]
Keepass (original) have more functionality to customize , I'll hope keepassxc will have equal or more, but now its not enought personally for me and this reason why I like keepass much more, but it is not officially native version for GNU Linux (( 
KeepassXc have a nice interface, more modern, but... first is functionality.
I often use auto-type with override URL to private, in keepass I can choose in Property of Entry, besides the fact that I can change auto-type sequence. 
Also Keepass have many interesting options in configuration to auto-lock after any actions.

Do you plan add CTRL+U (CTRL+SHIFT+U) with custom or Private URL tab?

Or if it is already realized, how can I use it same as in original Keepass client?

Thx

**Comment by droidmonkey** [comment]
Opening with private mode browser is in development. You can use it today by defining a custom command url. Read our user guide for more info
